gtsam/basis/Chebyshev2.h

```diff
@@ -0,0 +1,137 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file Chebyshev2.h
+ * @brief Chebyshev parameterizations on Chebyshev points of second kind.
```

Comment:
Propose replacing by

@brief Pseudo-spectral parameterization for Chebyshev polynomials of the second kind.

In a pseudo-spectral case, rather than the parameters acting as weights for the bases polynomials (as in Chebyshev2Basis), here the parameters are the *values* at a specific set of points in the interval, the “Chebyshev points”. These values uniquely determine the polynomial that interpolates them at the Chebyshev points.